lib/diagnostic.js

```diff
@@ -6,31 +6,69 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function () {
+  animals : 'Cat', 'Dog', 'Bird';
+  return normalize.[animals];
 };
+console.log(normalizeWords(/\w\S*/));
```

Comment:
You're not really on the right track here. Please study the solution and make an appointment for 1-1 if anything is still unclear.
```
const normalizeWords = function (text) {
  let words = text.split(/\s+/);
  for (let i = 0; i < words.length; i++) {
    let word = words[i];
    words[i] = word.replace(/\W+/, '').toUpperCase();
  }
  return words;
};
```